kamu-core/src/domain/query_service.rs

```diff
@@ -75,6 +82,16 @@ pub enum QueryError {
     ),
 }
 
+/////////////////////////////////////////////////////////////////////////////////////////
+
+#[derive(Error, Clone, PartialEq, Eq, Debug)]
+#[error("Dataset schema is not available yet: {dataset_ref}")]
```

Comment:
Nitpick: "not yet available" I think is more grammatically correct, but I don't have a proof :)

Reply:
Let it be